flow/src/org/labkey/flow/webparts/FlowSummary.jsp

```diff
@@ -115,29 +115,31 @@
     }
 </style>
 
-<script type="text/javascript">
+<script type="text/javascript" nonce="<%=getScriptNonce()%>">
     function createRenderer(detailsURL, countProperty, countLabel)
     {
+        const h = LABKEY.Utils.encodeHtml;
+
         return function (el, json) {
             var html = "<table border='0'>";
             for (var i = 0; i < <%=DISPLAY_MAX_ROWS%> && i < json.rows.length; i++) {
                 var row = json.rows[i];
                 var name = row.Name.value;
                 var url = row.Name.url;
-                var comment = row["Flag/Comment"].value ? " title='" + row["Flag/Comment"].value + "'" : "";
+                var comment = row["Flag/Comment"].value ? " title='" + h(row["Flag/Comment"].value) + "'" : "";

```

Comment:
Shouldn't this get q() too/instead?

Reply:
No.  LABKEY.Utils.encodeHtml() will remove any single-quote, double-quote, or backslash characters, so I believe this is correct.  In any event, q() is a server-side method.